api/indexer.oas2.json

```diff
@@ -917,6 +957,16 @@
         "is-frozen": {
           "description": "\\[f\\] whether or not the holding is frozen.",
           "type": "boolean"
+        },
+        "opted-in-at-round": {
+          "description": "Round during which this asset holding was opted in.",
+           "type": "integer",
+           "x-algorand-format": "uint64"
+        },
+        "opted-out-at-round": {
+          "description": "Round during which this asset holding was closed.",
```

Comment:
"Account was opted out from the asset" maybe? To keep it consistent with the previous one.